src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.impl.FileReaderImpl;
+import core.basesyntax.service.impl.FileWriterImpl;
+import core.basesyntax.service.impl.FruitShopServiceImpl;
+import core.basesyntax.service.impl.ParserImpl;
+import java.util.List;
+
+public class Main {
+    private static final String PATH = "src/main/java/core/basesyntax/db/data.csv";
+    private static final String REPORT = "src/main/java/core/basesyntax/db/report.csv";
```

Comment:
Better to use more precise naming
```suggestion
    private static final String DATA_SOURCE_PATH = "src/main/java/core/basesyntax/db/data.csv";
    private static final String REPORT_PATH = "src/main/java/core/basesyntax/db/report.csv";
```